disruption_py/machine/cmod/efit.py

```diff
@@ -130,14 +128,6 @@ def get_efit_parameters(params: PhysicsMethodParams):
                 print("unable to get v_loop_efit")
                 efit_data["v_loop_efit"] = np.full(len(efit_time), np.nan)
 
-            # Compute beta_n
```

Comment:
there was another comment about `beta_n` just before the `if` statement acting on pre-2000 shots.